cmd/upload.go

```diff
@@ -66,19 +69,72 @@ var uploadCmd = &cobra.Command{
 			attrs.WhoPaysForReads = wp // set given value
 		}
 
-		if len(thumbnailpath) > 0 {
-			if encrypt {
-				err = allocationObj.EncryptAndUploadFileWithThumbnail(localpath, remotepath, thumbnailpath, attrs, statusBar)
-			} else {
-				err = allocationObj.UploadFileWithThumbnail(localpath, remotepath, thumbnailpath, attrs, statusBar)
+		if stream {
+
+			chunkSize, _ := cmd.Flags().GetInt("chunksize")
+
+			fileReader, err := os.Open(localpath)
+			if err != nil {
+				goto ThrownErr
```

Comment:
Please don't use `goto. Find another way. 
For instance you could add the `ThrowErr`code to a `defer`  block and replace the  `goto ThrowErr`  with  `return`.